### ISSUE TITLE
Persist the Tabs using query parameters

### ISSUE THREAD

**Issue by rahulxdemon** [soummyaanon/-noteX#5]
**Persist the all notes and Favorites tab in the query parameters**
When we are in the `favorite` tab and reload the app then the tab changed to the `All notes` tab

**I would like to use the query parameters using react-router-dom**
We can use the built in hooks to update the url and persist the tabs

![image](https://github.com/user-attachments/assets/bd4c4f74-a1b9-4dab-b47d-90c6e7a483b6)

**Comment by soummyaanon** [comment]
IF you want To contribute I have assigned You the issue, You can Freely contribute to this project

**Comment by rahulxdemon** [comment]
I will start working on it

**Comment by rahulxdemon** [comment]
@soummyaanon can you tell me the env variables required for this project

There is not `.env.example` file to do so 🥲

**Comment by soummyaanon** [comment]
Just now I have added those env. examples sorry I totally forgot to add env examples

**Comment by rahulxdemon** [comment]
@soummyaanon no problem I will pull the changes

**Comment by rahulxdemon** [comment]
Where can I get the below credentials

```
VITE_APPWRITE_COLLECTION_ID="your_collection_id"
VITE_APPWRITE_USERS_COLLECTION_ID="your_users_collection_id"
```

Are they present in the app write dashboard

**Comment by soummyaanon** [comment]
> Where can I get the below credentials
> 
> ```
> VITE_APPWRITE_COLLECTION_ID="your_collection_id"
> VITE_APPWRITE_USERS_COLLECTION_ID="your_users_collection_id"
> ```
> 
> Are they present in the app write dashboard

Yes  like you have to create two collection in appwrite dashboard one is user and another is notes thats it
